src/universal/modules/landing/ducks/auth.js

```diff
@@ -0,0 +1,27 @@
+const SET_AUTH_TOKEN = 'notifications/SET_AUTH_TOKEN';
```

Comment:
I sense we're going to have a few login strategies that won't involve redirection to landing. If it's not too inconvenient, mind shifting it up to `redux`?

Also: love the socket handshake refresh. I think that'd be just rad.